src/docs/product/alerts/index.mdx

```diff
@@ -52,7 +52,7 @@ By default all users can mute alerts for everyone. Sentry users with manager or
 
 </Note>
 
-Issue alerts can be muted on the alert rule details page by clicking the "Mute" button. This will mute the alert from notifying you until you click "Unmute". If you want to mute the alert from firing entirely, select "Mute for everyone" from the dropdown.
+Issue alerts can be muted on the alert rule details page by clicking the "Mute" button. This will mute the alert from notifying you until you click "Unmute". If you want to mute the alert from firing entirely, select "Mute for everyone" from the dropdown. If the only alert action is set to notify an integration, the only option available will be "Mute for everyone". If there are multiple alert actions which notify an integration and notify a user or team and you select "Mute for me", the integration alert will still fire.
```

Comment:
I hate using "only" twice in one sentence but I was having a hard time rewording it.

Reply:
i don't mind it , i feel like there isn't a better way to phrase it.